### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by allenh1**
Releases version 0.3.0.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,16 +25,18 @@
...
-    author_email='hunter@openrobotics.org',
+    author_email='hunterlallen@protonmail.com',
     url='https://github.com/ros-infrastructure/superflore',
     keywords=['ROS'],
     install_requires=install_requires,
     python_requires='>=3',
     classifiers=['Programming Language :: Python',
-                 'License :: OSI Approved :: Apache Software License'],
+                 'License :: OSI Approved :: Apache Software License'
+                 'License :: OSI Approved :: MIT Software License'
```
Comment (allenh1) [inline setup.py]: @tfoote please check this line for me, I'm not srue how it is supposed to look

Comment (tfoote) [reply setup.py]: with the commas is looks good to me.

**Comment by allenh1** [comment]
@tfoote thanks for the review. I'm holding this back until #192 and #196 make it in. After that I'll regenerate the changelog again and we'll make the release.

**Comment by allenh1** [comment]
@tfoote Is the failing CI related to the build farm problems?

**Comment by tfoote** [comment]
The GPG key needs to be updated. https://discourse.ros.org/t/new-gpg-keys-deployed-for-packages-ros-org/9454

cross reference: https://github.com/ros-infrastructure/roswiki/issues/276

**Comment by allenh1** [comment]
@tfoote thanks -- filed #199 to fix it.

**Comment by allenh1** [comment]
@tfoote I'll ping you for a review here in a bit, but I'm going to PR two more things in before the release (nothing that should modify patch notes). Namely:

 * [ ] #200 
 * [ ] #201 
 * [ ] #198

**Comment by allenh1** [comment]
@tfoote friendly ping to look at the above PR's for me when you get a chance to

**Pull request merged**